<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1313</width>
    <height>933</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>-30</y>
      <width>261</width>
      <height>911</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
	alternate-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(122, 122, 122, 193), stop:1 rgba(255, 255, 255, 255));
font-family:century gothic;
font-size:24pix
}
QFrame
{
background:#333;
border-radius:15px;
}
#pushButton_2
{
color:white;
background:red;
border-radius:15px;
}

#pushButton_2:hover
{
color:red;
border-radius:15px;
background:#300;

}







QToolButton
{
background:red;
border-radius:60px;
}
QLabel{
color:white;

}
QLineEdit
{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;


}












</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* style.qss */

QPushButton {
    background-color: #e74c3c;
    border: 2px solid #c0392b;
    color: white;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #c0392b;
}

QPushButton:pressed {
    background-color: #922b21;
}

QPushButton:checked {
    background-color: #d35400;
}

QPushButton:checked:hover {
    background-color: #b03a2e;
}

QPushButton:checked:pressed {
    background-color: #943126;
}

QPushButton:disabled {
    background-color: #95a5a6;
    border: 2px solid #7f8c8d;
    color: #bdc3c7;
    cursor: not-allowed;
}
</string>
    </property>
    <property name="text">
     <string>Gestion de rapport</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>261</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* style.qss */

QPushButton {
    background-color: #e74c3c;
    border: 2px solid #c0392b;
    color: white;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #c0392b;
}

QPushButton:pressed {
    background-color: #922b21;
}

QPushButton:checked {
    background-color: #d35400;
}

QPushButton:checked:hover {
    background-color: #b03a2e;
}

QPushButton:checked:pressed {
    background-color: #943126;
}

QPushButton:disabled {
    background-color: #95a5a6;
    border: 2px solid #7f8c8d;
    color: #bdc3c7;
    cursor: not-allowed;
}
</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* style.qss */

QPushButton {
    background-color: #e74c3c;
    border: 2px solid #c0392b;
    color: white;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #c0392b;
}

QPushButton:pressed {
    background-color: #922b21;
}

QPushButton:checked {
    background-color: #d35400;
}

QPushButton:checked:hover {
    background-color: #b03a2e;
}

QPushButton:checked:pressed {
    background-color: #943126;
}

QPushButton:disabled {
    background-color: #95a5a6;
    border: 2px solid #7f8c8d;
    color: #bdc3c7;
    cursor: not-allowed;
}
</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* style.qss */

QPushButton {
    background-color: #e74c3c;
    border: 2px solid #c0392b;
    color: white;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #c0392b;
}

QPushButton:pressed {
    background-color: #922b21;
}

QPushButton:checked {
    background-color: #d35400;
}

QPushButton:checked:hover {
    background-color: #b03a2e;
}

QPushButton:checked:pressed {
    background-color: #943126;
}

QPushButton:disabled {
    background-color: #95a5a6;
    border: 2px solid #7f8c8d;
    color: #bdc3c7;
    cursor: not-allowed;
}
</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* style.qss */

QPushButton {
    background-color: #e74c3c;
    border: 2px solid #c0392b;
    color: white;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #c0392b;
}

QPushButton:pressed {
    background-color: #922b21;
}

QPushButton:checked {
    background-color: #d35400;
}

QPushButton:checked:hover {
    background-color: #b03a2e;
}

QPushButton:checked:pressed {
    background-color: #943126;
}

QPushButton:disabled {
    background-color: #95a5a6;
    border: 2px solid #7f8c8d;
    color: #bdc3c7;
    cursor: not-allowed;
}
</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>351</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
    font-family: century gothic;
    color: #000; /* Texte noir par défaut */
    border-color: #000; /* Couleur de la bordure noire par défaut */
    font-size: 14px;
}

#frame {
    font-family: century gothic;
    border: 1px solid #000; /* Bordure noire par défaut */
    border-radius: 30px;
    border-color: #eaeaea;
    margin-top: 0.5em;
    background: white;
}

QFrame[frameShape=&quot;4&quot;],
QFrame[frameShape=&quot;5&quot;] {
    border: none;
    border-bottom: 1px solid #000; /* Ligne noire par défaut */
}

QComboButton {
    font-family: century gothic;
    background: transparent;
    border: none;
    color: #000; /* Texte noir par défaut */
    border-bottom: 1px solid #000; /* Ligne noire par défaut */
}

QComboButton:hover {
    color: red; /* Change le texte en rouge au survol */
    border-bottom: 1px solid red; /* Change la ligne en rouge au survol */
}

QLabel {
    background: transparent;
}

QLineEdit,
QDateTimeEdit {
    font-family: century gothic;
    background: transparent;
    border: none;
    color: #000; /* Texte noir par défaut */
    border-bottom: 1px solid #000; /* Ligne noire par défaut */
}

QLineEdit:hover,
QDateTimeEdit:hover {
    color: red; /* Change le texte en rouge au survol */
    border-bottom: 1px solid red; /* Change la ligne en rouge au survol */
}</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>id</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>73</y>
       <width>47</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>rapport</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>date</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>id university</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>193</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>idemployee</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>194</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>420</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* style.qss */

QPushButton {
    background-color: #e74c3c;
    border: 2px solid #c0392b;
    color: white;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #c0392b;
}

QPushButton:pressed {
    background-color: #922b21;
}

QPushButton:checked {
    background-color: #d35400;
}

QPushButton:checked:hover {
    background-color: #b03a2e;
}

QPushButton:checked:pressed {
    background-color: #943126;
}

QPushButton:disabled {
    background-color: #95a5a6;
    border: 2px solid #7f8c8d;
    color: #bdc3c7;
    cursor: not-allowed;
}
</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>420</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* style.qss */

QPushButton {
    background-color: #e74c3c;
    border: 2px solid #c0392b;
    color: white;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #c0392b;
}

QPushButton:pressed {
    background-color: #922b21;
}

QPushButton:checked {
    background-color: #d35400;
}

QPushButton:checked:hover {
    background-color: #b03a2e;
}

QPushButton:checked:pressed {
    background-color: #943126;
}

QPushButton:disabled {
    background-color: #95a5a6;
    border: 2px solid #7f8c8d;
    color: #bdc3c7;
    cursor: not-allowed;
}
</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>290</y>
       <width>113</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>360</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="rfid">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>510</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>671</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
    font-family: century gothic;
    color: #000; /* Texte noir par défaut */
    border-color: #000; /* Couleur de la bordure noire par défaut */
    font-size: 14px;
}

#frame {
    font-family: century gothic;
    border: 1px solid #000; /* Bordure noire par défaut */
    border-radius: 30px;
    border-color: #eaeaea;
    margin-top: 0.5em;
    background: white;
}

QFrame[frameShape=&quot;4&quot;],
QFrame[frameShape=&quot;5&quot;] {
    border: none;
    border-bottom: 1px solid #000; /* Ligne noire par défaut */
}

QComboButton {
    font-family: century gothic;
    background: transparent;
    border: none;
    color: #000; /* Texte noir par défaut */
    border-bottom: 1px solid #000; /* Ligne noire par défaut */
}

QComboButton:hover {
    color: red; /* Change le texte en rouge au survol */
    border-bottom: 1px solid red; /* Change la ligne en rouge au survol */
}

QLabel {
    background: transparent;
}

QLineEdit,
QDateTimeEdit {
    font-family: century gothic;
    background: transparent;
    border: none;
    color: #000; /* Texte noir par défaut */
    border-bottom: 1px solid #000; /* Ligne noire par défaut */
}

QLineEdit:hover,
QDateTimeEdit:hover {
    color: red; /* Change le texte en rouge au survol */
    border-bottom: 1px solid red; /* Change la ligne en rouge au survol */
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>280</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* style.qss */

QPushButton {
    background-color: #e74c3c;
    border: 2px solid #c0392b;
    color: white;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #c0392b;
}

QPushButton:pressed {
    background-color: #922b21;
}

QPushButton:checked {
    background-color: #d35400;
}

QPushButton:checked:hover {
    background-color: #b03a2e;
}

QPushButton:checked:pressed {
    background-color: #943126;
}

QPushButton:disabled {
    background-color: #95a5a6;
    border: 2px solid #7f8c8d;
    color: #bdc3c7;
    cursor: not-allowed;
}
</string>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>491</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Styles for QTableView */
QTableView {
    /* Keep the default background color */
    border: 2px solid #2980b9;
    color: black; /* Set the text color to black */
}

QTableView::item {
    border: 1px solid red; /* Set the border color for items to red */
    padding: 6px;
}

QTableView::item:selected {
    background-color: #e74c3c; /* Change only the selected item color to red */
    color: white;
}
</string>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>280</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* style.qss */

QPushButton {
    background-color: #e74c3c;
    border: 2px solid #c0392b;
    color: white;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #c0392b;
}

QPushButton:pressed {
    background-color: #922b21;
}

QPushButton:checked {
    background-color: #d35400;
}

QPushButton:checked:hover {
    background-color: #b03a2e;
}

QPushButton:checked:pressed {
    background-color: #943126;
}

QPushButton:disabled {
    background-color: #95a5a6;
    border: 2px solid #7f8c8d;
    color: #bdc3c7;
    cursor: not-allowed;
}
</string>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    background-color: #e74c3c;
    border: 2px solid #c0392b;
    color: white;
    padding: 5px;
    font-size: 14px;
    border-radius: 5px;
    min-width: 100px;
}

QComboBox:hover {
    background-color: #c0392b;
}

QComboBox:pressed {
    background-color: #922b21;
}

QComboBox:disabled {
    background-color: #95a5a6;
    border: 2px solid #7f8c8d;
    color: #bdc3c7;
    cursor: not-allowed;
}</string>
     </property>
     <item>
      <property name="text">
       <string>Par defaut</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>id</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nom</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>210</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rechh">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>350</y>
      <width>521</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_78">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>50</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>160</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* style.qss */

QPushButton {
    background-color: #e74c3c;
    border: 2px solid #c0392b;
    color: white;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #c0392b;
}

QPushButton:pressed {
    background-color: #922b21;
}

QPushButton:checked {
    background-color: #d35400;
}

QPushButton:checked:hover {
    background-color: #b03a2e;
}

QPushButton:checked:pressed {
    background-color: #943126;
}

QPushButton:disabled {
    background-color: #95a5a6;
    border: 2px solid #7f8c8d;
    color: #bdc3c7;
    cursor: not-allowed;
}
</string>
     </property>
     <property name="text">
      <string>Envoyer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>331</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="output">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>481</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Identifiant</string>
     </property>
    </widget>
    <zorder>label_16</zorder>
    <zorder>label_78</zorder>
    <zorder>pushButton_10</zorder>
    <zorder>input</zorder>
    <zorder>output</zorder>
   </widget>
   <widget class="QLabel" name="chartview">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>570</y>
      <width>561</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1313</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
